Library/Formula/terraform.rb

```diff
@@ -3,64 +3,68 @@
 
 class Terraform < Formula
   homepage "https://www.terraform.io/"
-  url "https://github.com/hashicorp/terraform/archive/v0.5.0.tar.gz"
-  sha1 "6befb71e505efa246cb48e9ece829b93fea367fb"
+  url "https://github.com/hashicorp/terraform/archive/v0.5.1.tar.gz"
+  sha1 "7df99086b2ef218c0a56841dbd43a054b19903fb"
 
   bottle do
     cellar :any
-    sha256 "ebf80d6dadaac89c0492030a44ae6ae3c386754fe467cf705c01a0334cded4ef" => :yosemite
-    sha256 "5816bc8ad31ee155e9783b948f297bca86096189f65c13b3e66ee3c138d29e45" => :mavericks
-    sha256 "7c2ef7e1bb3ae1da851d05d0d67c7d86f155b0fc4f6b8b03af844063e8c891d3" => :mountain_lion
+  end
+
+  # patch from upstream, will be included in next release
+  # fix regression in networkacl with conflicts with
+  patch do
+    url "https://github.com/hashicorp/terraform/commit/af09f257c8d5522ce78f9a9088d04f653fb73ec9.patch"
```

Comment:
For reference Github isn't appending git version numbers to `.patch` files anymore - these should be safe to use now.